### PR TITLE
Update ovms with gpu support and remove the other template

### LINKED ISSUE #1989: [Story]: Add new OVMS OOTB template
### Goal

Implement new ovms image and deprecate the GPU one

### Dependency issue

No dependencies

### Itemized goals

* Figure out how to remove the gpu OOTB for already deployed servers (it appears it should be done by the operator automatically)
* Change the image from the OVMS template based on the [latest template](https://quay.io/repository/opendatahub/openvino_model_server?tab=history) 
* Remove the gpu limitation from the OVMS template

### Aspects continued elsewhere

_No response_

### Mocks

No mocks

### PR DESCRIPTION
**Pull request by lucferbux**
## Description

Closes #1989

## How Has This Been Tested?

1. Delete the old templates
2. Deploy the new templates
3. Create a new ovms model with the new Custom Serving Runtime
4. **NOT TESTED IN A GPU CLUSTER, WE SHOULD DO THAT AT ONE POINT BEFORE THE RELEASE**

## Test Impact

Not needed, manifest files, we'll might include something for e2e testing in the future.

## Request review criteria:

Self checklist (all need to be checked):
- [X] The developer has manually tested the changes and verified that the changes work
- [X] Commits have been squashed into descriptive, self-contained units of work (e.g. 'WIP' and 'Implements feedback' style messages have been removed)
- [X] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [X] The developer has added tests or explained why testing cannot be added (unit tests & storybook for related changes)

If you have UI changes: 

- [ ] Included any necessary screenshots or gifs if it was a UI change.
- [ ] Included tags to the UX team if it was a UI/UX change (find relevant UX in the [SMEs](https://github.com/opendatahub-io/odh-dashboard/tree/main/docs/smes.md) section).

After the PR is posted & before it merges:
- [ ] The developer has tested their solution on a cluster by using the image produced by the PR to `main`

### CODE REVIEW

**Comment by lucferbux** [comment]
@Xaenalt can you take a look the spec is correct please?

File: manifests/modelserving/ovms-ootb.yaml [hunk 1]
```diff
@@ -16,11 +16,13 @@ objects:
...
       annotations:
         openshift.io/display-name: 'OpenVINO Model Server'
-        opendatahub.io/disable-gpu: 'true'
       labels:
         opendatahub.io/dashboard: 'true'
     spec:
       builtInAdapter:
+        env:
+          - name: OVMS_FORCE_TARGET_DEVICE
+            value: NVIDIA
```
Comment (Xaenalt) [inline manifests/modelserving/ovms-ootb.yaml:25]: This should be AUTO per https://docs.openvino.ai/2023.0/notebooks/106-auto-device-with-output.html for supporting both paths

File: manifests/modelserving/kustomization.yaml [hunk 2]
```diff
@@ -5,12 +5,11 @@ commonLabels:
   app.kubernetes.io/part-of: odh-dashboard
 resources:
   - ovms-ootb.yaml
-  - ovms-gpu-ootb.yaml
   - caikit-ootb.yaml
 images:
   - name: ovms-1
     newName: quay.io/opendatahub/openvino_model_server
-    digest: sha256:20dbfbaf53d1afbd47c612d953984238cb0e207972ed544a5ea662c2404f276d
+    digest: sha256:sha256:2cbe8a48ab0bc6fe7fb76919bf33253e83a6218a9c4b486b744c3dcf30679616
```
Comment (Xaenalt) [inline manifests/modelserving/kustomization.yaml:12]: Extra sha256 here, pretty sure that'll be a syntax error

**Review by Xaenalt — CHANGES_REQUESTED** [review]

File: manifests/modelserving/kustomization.yaml (hunk 2 above)
Comment (lucferbux) [reply manifests/modelserving/kustomization.yaml:12]: Agh, damn, that happens cause I gave format to the file, i tested it first quickly to see if the image was working, and then i misscopied the second time, good catch!

**Review by Xaenalt — APPROVED** [review]

**Comment by Xaenalt** [comment]
A note we may need to bump the image version again later for that CVE that's been going around

**Pull request merged**